Assets/Editor/Tests/UnarchiverTest.cs

```diff
@@ -111,25 +111,25 @@ public void ProgressReporting()
 
             if (lastEntry.HasValue)
             {
-                Assert.AreEqual(lastEntry + 1, entry, "Entries are not following each other.");
+                Assert.IsTrue(lastEntry <= entry, "Entries are not following each other.");
```

Comment:
It seems that NUnit supports a very nice descriptive assertion model.

The equivalent assertion would look something like this:
```csharp
Assert.That(lastEntry, Is.LessThanOrEqualTo(entry));
```

Wouldn't this new model be more appropriate?

Reply:
Wow, I didn't know about this!

I'm going to start using it, thanks! ;)